src/Uno.UI/UI/Xaml/Controls/ItemsControl/ItemsControl.cs

```diff
@@ -85,6 +85,7 @@ void Initialize()
 			OnDisplayMemberPathChangedPartial(string.Empty, this.DisplayMemberPath);
 
 			_items.VectorChanged += (s, e) => {
+				OnItemsChanged(null);
```

Comment:
should the parameter be null?

Reply:
Documentation says that the value is implementation specific, and is effectively null in the current runtime.